app/src/main/res/values/styles.xml

```diff
@@ -19,4 +19,8 @@
         <item name="android:background">?attr/colorPrimary</item>
     </style>
 
+    <style name="PopupMenu" parent="@style/Widget.AppCompat.Light.PopupMenu">
```

Comment:
You also need to set the theme's popup menu style to make this effective. So in the App Theme, also add:
```
<item name="popupMenuStyle">@style/PopupMenu</item>
```